helpers/dbload.test.js

```diff
@@ -0,0 +1,34 @@
+jest.mock('sequelize')
+const defaultOptions = {
+  host: process.env.DB_HOST || 'city',
+  logging: false,
+  dialect: 'postgres',
+  pool: {
+    max: 5,
+    min: 0,
+    acquire: 30000,
+    idle: 10000
+  }
+}
+describe('dbload tests', () => {
+  beforeEach(() => {
```

Comment:
This should work so that you don't have to export `options` from dbload

```
let defaultOptions
beforeEach(() => {
    jest.resetModuleRegistry()
    defaultOptions = {
        host: process.env.DB_HOST || 'city',
        logging: false,
        dialect: 'postgres',
        pool: {
            max: 5,
            min: 0,
            acquire: 30000,
            idle: 10000
        }
        port: ((process.env.DB_PORT as unknown) as number) || 5432
    }
})
```